client/state/stats/lists/utils.js

```diff
@@ -14,3 +16,24 @@ import toPairs from 'lodash/toPairs';
 export function getSerializedStatsQuery( query = {} ) {
 	return JSON.stringify( sortBy( toPairs( query ), ( pair ) => pair[ 0 ] ) );
 }
+
+export const Parsers = {
```

Comment:
>Based on our variable naming guidelines:

I'd be to blame for the suggestion. I agree it's not one that's been documented, but has been used in the past (maybe mostly by me 😄  [example](https://github.com/Automattic/wp-calypso/blob/master/client/lib/media/constants.js)). I guess it'd be hard to classify this usage (objects of constants?) so perhaps best to change to your alternatives.